opentreemap/treemap/templates/treemap/map.html

```diff
@@ -7,7 +7,7 @@
 {% endblock head_extra %}
 
 {% block searchscripts %}
-<script src="//maps.google.com/maps/api/js?v=3&sensor=false"></script>
+<script src="https://maps.googleapis.com/maps/api/js?key={{ settings.GOOGLE_MAPS_API_KEY }}"></script>
```

Comment:
Fixed this, in a rebased commit for this PR (5928556) and with a new PR https://github.com/OpenTreeMap/otm-addons/pull/1202